platform-sdk/swirlds-virtualmap/src/test/java/com/swirlds/virtualmap/internal/pipeline/VirtualPipelineTests.java

```diff
@@ -812,57 +812,68 @@ void mergeReleaseRace() throws InterruptedException {
     }
 
     /**
-     * Measure the time that it takes to make another copy, assert that it is within 10ms of the expected time.
+     * Make a copy and calculate the flush backpressure pause, assert that the value equals to the expected time.
      */
-    private static void copyAndMeasureTime(final Deque<DummyVirtualRoot> copies, final int expectedTimeMs) {
-        final Instant start = Instant.now();
-
+    private static void copyAndAssertFlushBackpressurePause(
+            final Deque<DummyVirtualRoot> copies, final int expectedTimeMs) {
         final DummyVirtualRoot copy = copies.getLast().copy();
         copies.add(copy);
+        final Duration duration = copy.getPipeline().calculateFlushBackpressurePause();
+        if (duration == null) {
+            // no backpressure applied
+            return;
+        }
 
-        final Instant end = Instant.now();
-        final Duration duration = Duration.between(start, end);
-
-        final int ms = (int) duration.toMillis();
+        assertEquals(expectedTimeMs, duration.toMillis());
```

Comment:
Minor: previously there was a small range of values allowed and now we're expecting a precise match here. I think this may introduce a degree of flakiness, now or in the future when the tests run on a different hardware. I'd still allow a range here. The comment applies to a test below as well.

Reply:
No, it can't introduce any flakiness because these numbers are strictly deterministic. This was the whole point of the refactoring.

Reply:
Sounds good.